src/main/features/claim/routes/pay.ts

```diff
@@ -105,6 +108,12 @@ async function successHandler (res, next) {
       )
     }
   }
+  const payClient: PayClient = await getPayClient(req)
+  const paymentReference = draft.document.claimant.payment.reference
+
+  const ccdCaseNumber = savedClaim.ccdCaseId === undefined ? 'UNKNOWN' : String(savedClaim.ccdCaseId)
+
+  await payClient.update(user, paymentReference, savedClaim.externalId, ccdCaseNumber)
```

Comment:
Do we not have to wait for reply before redirecting? what happens if this call is not successful?